src/java.base/share/classes/java/lang/UnsupportedOperationException.java

```diff
@@ -37,13 +37,15 @@
  */
 public class UnsupportedOperationException extends RuntimeException {
     /**
-     * Constructs an UnsupportedOperationException with no detail message.
+     * Constructs a new {@code UnsupportedOperationException} with {@code null} as its
+     * detail message. The cause is not initialized, and may subsequently be
+     * initialized by a call to {@link #initCause(Throwable)}.
      */
     public UnsupportedOperationException() {
     }
 
     /**
-     * Constructs an UnsupportedOperationException with the specified
+     * Constructs a new {@code UnsupportedOperationException} with the specified
      * detail message.
```

Comment:
I think it makes sense to expand the description of this constructor with `“The cause is not initialized, and may subsequently be initialized by a call to {@link #initCause(Throwable)}.”` as per no-arg constructor.